src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ssl.c

```diff
@@ -4,6 +4,16 @@
 
 #include "pal_ssl.h"
 
+// This would come from private header  <Security/SecureTransportPriv.h>
+// 10.13.4 introduced public API but linking would fail on all prior versions.
+// This can be revisited after we drop support for 10.12.
+// Also note, this works only on client side in coretls-155 and lower.
+
+const void * SSLGetALPNData(SSLContextRef contex, size_t *length);
```

Comment:
I feel like `clang-format` would have written this as `void*`, not `void *`.  I *think* this file is already otherwise clang-format formatted.